routes/index.js

```diff
@@ -124,6 +125,24 @@ router.get('/login', (req, res) => {
 						);
 					}
 
+					let accountRecovered = false;
+
+					if (Number.isInteger(user.deletion_interval_in_days)) {
+						await recoverUserAccount(user.id).catch(
+							(err) => {
+								return serverErrorHandler(res, 'Error: Failed to recover user account in login', err, { user_account_recovery_failed: true });
+							}
+						);
+
+						await user.update({ deletion_interval_in_days: null }).catch(
```

Comment:
You update this outside of the recover function just to see if it errs out or not?